test/gen_casadi_test.py

```diff
@@ -658,7 +658,7 @@ def test_delay_for_loop_with_expand_vectors(self):
         ref_model = Model()
 
         def _array_mx(name, n):
-            return np.array([ca.MX.sym("{}[{}]".format(name, i)) for i in range(n)])
+            return np.array([ca.MX.sym("{}[{}]".format(name, i+1)) for i in range(n)])
```

Comment:
should be `i + 1` tho I am not sure we care at this point

Reply:
We have `i+1` in `model.py`, so in a sense it's consistent. If I'm going to change this, I'd have to change the formatting of those as well.

Reply:
sure, ok. tox will complain tho I think.